package-build/package-build.el

```diff
@@ -146,6 +147,10 @@ function for access to this function")
     (:exclude ".dir-locals.el" "test.el" "tests.el" "*-test.el" "*-tests.el"))
   "Default value for :files attribute in recipes.")
 
+(defun mappend (fn seq)
+  "Apply `FN' to the `SEQ', and append results."
+  (apply #'append (mapcar fn seq)))
+
```

Comment:
yes i totally ignored that lack of namespace thing. 
it seems it's safe to use cl-mapcan there.

- what do you think about a separate doc-folder option in recipe spec ? 
(ECB requires customization after install, because it's searching info file under info-help.)